roundabout-core/src/test/java/at/fhv/itm3/s2/roundabout/entity/StreetSectionTest.java

```diff
@@ -337,7 +337,7 @@ public void updateAllCarsPositions_oneCar() {
             5,
             1,
             4,
-            1.5
+            1.5, 1
         );
```

Comment:
i know you didnt write this, but a constructor with a lot of numbers is quite unreadable as we need to check the class first to understand each value